requirements/tutorial.txt

```diff
@@ -1,4 +1,4 @@
+# Generated via tools/generate_requirements.py and pre-commit hook.
+# Do not edit this file; modify pyproject.toml instead.
 jupyter>=1.0
-matplotlib>=3.3
-seaborn>=0.10
-ipython<=8.12.0
\ No newline at end of file
+ipython<=8.12.0
```

Comment:
matplotlib not needed anymore because already in the default?

Reply:
Correct, it was redundant before.